python/MaterialXTest/tests_to_html.bat

```diff
@@ -1 +1 @@
-python tests_to_html.py -i ../../build
+python tests_to_html.py -i ../../build --destlang2 mdl
```

Comment:
Probably shouldn't hardcode this to mdl :).

Reply:
depends, I did not change the default in which GLSL is compared against OSL. MDL is the only option that makes sense I guess. But it's your decision. It's only a suggestion. What do you prefer as default:
* GLSL vs. OSL as before my changes
* GLSL vs. OSL vs. MDL when passing no arguments to the python script I think that makes two-way comparison odd because you would need to disable the third
* or my suggestion which is sort of a mixture of both

Reply:
Actually, I'd be more than happy to just delete this file. 
It's using a hard-coded location and is not part of any tests nor in CI.
(Also, we have Metal coming so it's option 4 :)).

Reply:
@kwokcb I believe this script is still the main path for render comparisons, and it's what Niklas and I still use to validate new work.

Reply:
I don't think the Window batch file is needed though is it? I just run the python script directly.

Reply:
I know that I still use it!  :)

Reply:
Ah ok. Sorry about that. I think defaulting to 2 way comparison of glsl vs osl then. Kai, this allows for vulcan, essl, metal to be a possible third option. Does this sound okay?

Reply:
From my point, you can drop `--destlang2 mdl` in the bat. I don't mind. Since Jonathan uses the script i would make it his decision.